### ISSUE TITLE
Can Tensorflow (not lite) run on Android NDK C++

### ISSUE THREAD

**Issue by jefhai** [tensorflow/tensorflow#24208]
It looks like features like Dropout are not supported on Tensorflow Lite + Android NDK.

I've got a Keras model that uses GPU, Dropout, CuDnnLSTM layer and I want to convert it to a tflite file or run it on Tensorflow for Android.

Is this possible? Is there a guide that tells me what is and is not supported on tflite?

**Comment by ymodak** [comment]
Yes. Here is an example of using TensorFlow from C++ on Android. 
https://github.com/tensorflow/tensorflow/tree/master/tensorflow/tools/benchmark

**Comment by ymodak** [comment]
Closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
